src/Microsoft.ML.AutoML.Interactive/Microsoft.ML.AutoML.Interactive.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Do you need to target .NET6 or can you target .NET3.1 so more users can use it?

Reply:
.NET 6:

https://github.com/plotly/Plotly.NET/blob/dev/src/Plotly.NET.CSharp/Plotly.NET.CSharp.csproj#L4